<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form</class>
 <widget class="QDialog" name="form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QGraphicsView" name="graphicsView">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>1081</width>
     <height>781</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="rollDiceButton">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>680</y>
     <width>100</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Roll Dice</string>
   </property>
  </widget>
  <widget class="QLabel" name="playerInfoLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>630</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Player Info Label:</string>
   </property>
  </widget>
  <widget class="QLabel" name="diceResultLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Dice Result:</string>
   </property>
  </widget>
  <widget class="QWidget" name="rightWidget" native="true">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>590</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
